tests/integration/pilot/vm/README.md

```diff
@@ -0,0 +1,79 @@
+# Writing Tests for VMs
+
+Support for including VMs in the integration tests has been added to the framework.
```

Comment:
Remove this sentence?

It seems redundant. When a developer finds this doc, he will assume this is already true.

Reply:
Removed.